amethyst_rendy/src/camera.rs

```diff
@@ -925,4 +925,55 @@ mod tests {
     fn orthographic_project_cube_off_centered_rotated() {
         unimplemented!()
     }
+
+    #[cfg(not(feature = "nightly"))]
+    #[bench]
+    pub fn transform_global_view_matrix_1000(b: &mut bencher::Bencher) {
```

Comment:
Could we put these into the `benches` dir so we don't need to special case stable Rust?

Reply:
Done. moved it to benches and swapped to Criterion